packs/data/extinction-curse-bestiary.db/gahlepod.json

```diff
@@ -63,7 +153,41 @@
                 "requirements": {
                     "value": ""
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "key": "Note",
+                        "outcome": [
+                            "criticalSuccess"
+                        ],
+                        "predicate": [
+                            "item:jaws",
+                            "churning-frenzy"
+                        ],
+                        "selector": "strike-damage",
+                        "text": "The gahlepod is spurred into a frenzy by the others. For the rest of the round, their jaws Strike deals @Localize[PF2E.PersistentDamage.Bleed1d6.criticalSuccess] in addition to the listed damage.",
+                        "title": "Churning Frenzy"
+                    },
+                    {
+                        "domain": "damage",
+                        "key": "RollOption",
+                        "option": "churning-frenzy",
+                        "toggleable": true,
+                        "value": false
+                    },
+                    {
+                        "key": "Note",
+                        "outcome": [
+                            "success"
+                        ],
+                        "predicate": [
+                            "item:jaws",
+                            "churning-frenzy"
+                        ],
+                        "selector": "strike-damage",
+                        "text": "The gahlepod is spurred into a frenzy by the others. For the rest of the round, their jaws Strike deals @Localize[PF2E.PersistentDamage.Bleed1d6.success] in addition to the listed damage.",
+                        "title": "Churning Frenzy"
```

Comment:
```suggestion
                    {
                        "key": "Note",
                        "outcome": [
                            "criticalSuccess"
                        ],
                        "predicate": [
                            "weapon:slug:jaws",
                            "churning-frenzy"
                        ],
                        "selector": "strike-damage",
                        "text": "PF2E.PersistentDamage.Bleed1d6.criticalSuccess,
                        "title": "{item|name}"
                    },
                    {
                        "domain": "damage",
                        "key": "RollOption",
                        "option": "churning-frenzy",
                        "toggleable": true,
                        "value": false
                    },
                    {
                        "key": "Note",
                        "outcome": [
                            "success"
                        ],
                        "predicate": [
                            "weapon:slug:jaws",
                            "churning-frenzy"
                        ],
                        "selector": "strike-damage",
                        "text": "PF2E.PersistentDamage.Bleed1d6.success",
                        "title": "{item|name}"
```

Reply:
I'll apply this manually since the trailing quotation mark is missing on the text property for Critical Success in the suggestion.